File: 6/TODO/3/numbers.c
// Implements linear search for numbers

#include <cs50.h>
#include <stdio.h>

int main(void)
{
    // An array of numbers
    int numbers[] = {4, 8, 15, 16, 23, 42};

    // Search for 50
    for (int i = 0; i < 6; i++)
    {
        if (numbers[i] == 50)
        {
            printf("Found\n");
            return 0;
        }
    }
    printf("Not found\n");
    return 1;
}

File: 6/TODO/3/numbers.py
# Implements linear search for numbers

import sys

# A list of numbers
numbers = [4, 8, 15, 16, 23, 42]

# Search for 50
if 50 in numbers:
    print("Found")
    sys.exit(0)
print("Not found")
sys.exit(1)
